src/confd/src/ietf-system.c

```diff
@@ -45,6 +45,12 @@ static char   *os  = NULL;
 static char   *nm  = NULL;
 static char   *id  = NULL;
 
+static const char *admin_groups[] = {
+	"wheel",
+	"frrvty",
```

Comment:
```suggestion
#ifdef HAVE_FRR
	"frrvty",
#endif
```
As this list grows, having this type of structure might make it easier to keep track of which deps pull in which groups. It would also avoid the need for `group_exists()`. Just a thought.

Now that I think about it: Is there a `podman` group that we should add for builds with container support?

Reply:
Good point. I'll add that to my commit 😀👍

Not that I've seen, but I'll have a look again at podman. I'm still at it with the other related issues anyway.